Lesson_1/Task1/Task1.3/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
Nice code style, but you should read from the console, not initilalize it by yourself